doc/source/tutorial/stats/sampling_pinv.rst

```diff
@@ -0,0 +1,271 @@
+.. _sampling-pinv:
+
+Polynomial interpolation based INVersion of CDF (PINV)
+======================================================
+
+.. currentmodule:: scipy.stats
+
+* Required: PDF
+* Optional: CDF, mode, center
+* Speed:
+
+  * Set-up: (very) slow
+  * Sampling: (very) fast
+
+Polynomial interpolation based INVersion of CDF (PINV) is an inversion method
+that only requires the density function to sample from a distribution. It is
+based on Polynomial interpolation of the PPF and Gauss-Lobatto integration
+of the PDF. It provides control over the interpolation error and integration
+error. Its primary purpose is to provide very fast sampling which is nearly
+the same for any given distribution at the cost of moderate to slow setup
+time. It is the fastest known inversion method for the fixed-parameter case.
+
+The inversion method is the simplest and most flexible to sample nonuniform
+random variates. For a target distribution with CDF :math:`F` and a uniform
+random variate :math:`U` sampled from :math:`\text{Uniform}(0, 1)`, a random
+variate X is generated by transforming the uniform random variate :math:`U`
+using the PPF (inverse CDF) of the distribution:
+
+.. math::
+    X = F^{-1}(U)
+
+This method is suitable for stochastic simulations because of its advantages.
+Some of the most attractive are:
+
+* It preserves the structural properties of the uniform random number sampler.
+* Transforms a uniform random variate :math:`U` one-to-one into non-uniform
+  random variates :math:`X`.
+* Easy and efficient sampling from truncated distributions.
+
+Unfortunately, the PPF is rarely available in closed form or too slow when
+available. For many distributions, the CDF is also not easy to obtain. This
+method addresses both the shortcomings. The user only has to provide the PDF
+and optionally a point near the mode (called "center") together with the size
+of the maximal acceptable error. It uses a combination of an adaptive and a
+simple Gauss-Lobatto quadrature to obtain the CDF and Newton's interpolation
+to obtain the PPF. The method is not exact, as it only produces random
+variates of the approximated distribution. Nevertheless, the maximal tolerated
+approximation error can be set close to the machine precision and its relative
+error is not much larger than :math:`10^{-15}`. The concept of u-error is used
+to measure and control the error. It is defined as:
```

Comment:
TBD if we should also describe the u-error in the general tutorial since it is also relevant for HINV.